demo/apisix/conf/apisix.yaml

```diff
@@ -105,4 +127,11 @@ plugin_configs:
         status_code: 200
         body: "APISIX already up!\n"  #For test, TODO remove
     desc: "response-rewrite"
+  -
+    id: limit-req-plugin
```

Comment:
Should we rename it to another, eg: `jmap-plugin`
We will have another config for another plugin. E.g: circuit breaker for this id. 
With the syntax above, not sure `plugin_config_id` will accept several ids inline,

Reply:
i agree it's better to regroup plugins common to all routes under one id. I'm ok with your suggestion :)